docs/Configuration.md

```diff
@@ -269,7 +296,8 @@ Do not use it (keep the default value "0"), or use it with care.
 
 ### number_of_processes_for_frontend_testing
 
-A positive integer.
+A positive integer. Max length 9 digits.
```

Comment:
This is extreme. Can run that many processes? I think limiting to 4 digits gives enought room.

Reply:
I thought of that as well. Having fewer ranges to choose from makes it easier on the implementation side though.